### PR TITLE
JETUT-73: Rewrite Jakarta Servlet introduction

### PR DESCRIPTION
**Pull request by BalusC**
Rewrote only the first 3 adocs due to budget cap.

### CODE REVIEW

**Comment by kito99** [comment]
@BalusC do you think we can merge this as-is (assuming we merge it with the Antora version). It seems to me that the changes are minor enough that they will still work with the parts of the chapter you didn't get a chance to edit.

**Comment by BalusC** [comment]
> @BalusC do you think we can merge this as-is (assuming we merge it with the Antora version). It seems to me that the changes are minor enough that they will still work with the parts of the chapter you didn't get a chance to edit.

Hmm, doable. Let me merge and missing link to new servlets019 adoc file.

**Comment by BalusC** [comment]
@kito99, ready for review now.

File: src/main/antora/modules/web/pages/servlets/servlets003.adoc [hunk 1]
```diff
@@ -1,58 +1,158 @@
...
-
-=== Using Scope Objects
-
-Collaborating web components share information by means of objects that are maintained as attributes of four scope objects.
-You access these attributes by using the `getAttribute` and `setAttribute` methods of the class representing the scope.
-<<_scope_objects>> lists the scope objects.
-
-[[_scope_objects]]
+Web components, like most objects, usually work with data to accomplish their tasks.
+Web components can store this information in a database or in a scoped bean.
```
Comment (kito99) [inline src/main/antora/modules/web/pages/servlets/servlets003.adoc:4]: "This data can be stored in a database or a scoped bean." (to about reusing "Web Components").

Shouldn't we also mention other types of objects, though?

File: src/main/antora/modules/web/pages/servlets/servlets003.adoc [hunk 2]
```diff
@@ -1,58 +1,158 @@
...
+`@jakarta.enterprise.context.SessionScoped`::
+In a servlet container, this is basically mapped to the `jakarta.servlet.http.HttpSession`.
+It lives for as long as the client is interacting with the web application with the same browser instance, and the session hasn't timed out at the server side.
+It is shared among all requests in the same session.
+
+`@jakarta.enterprise.context.ApplicationScoped`::
+In a servlet container, this is basically mapped to the `jakarta.servlet.ServletContext`.
+It lives for as long as the web application lives.
+It is shared among all requests in all sessions.
+
```
Comment (kito99) [inline src/main/antora/modules/web/pages/servlets/servlets003.adoc:28]: Are there scenarios where these aren't mapped as described? If not, removing "basically" would make sense.

**Review by kito99 — CHANGES_REQUESTED** [review]

File: src/main/antora/modules/web/pages/servlets/servlets003.adoc (hunk 2 above)
Comment (arjantijms) [reply src/main/antora/modules/web/pages/servlets/servlets003.adoc:28]: Might be better to remove "basically" here. In a servlet container that's the mapping.

File: src/main/antora/modules/web/pages/servlets/servlets.adoc [hunk 3]
```diff
@@ -1,8 +1,7 @@
-= Jakarta Servlet Technology
+= Jakarta Servlet
 
-include::ROOT:partial$not_updated.adoc[]
-
-Jakarta Servlet technology provides dynamic, user-oriented content in web applications using a request-response programming model.
+Jakarta Servlet is a corner stone web framework that can act as a presentation-oriented as well as a service-oriented web application.
+Jakarta Servlet intends to reduce the boilerplate code needed to convert the HTTP request into a Java object and to offer a Java object as a HTTP response, and to manage all the lifecycle around them.
```
Comment (arjantijms) [inline src/main/antora/modules/web/pages/servlets/servlets.adoc:4]: a HTTP -> an HTTP?

The H is pronounced as a vowel here.

File: src/main/antora/modules/web/pages/servlets/servlets003.adoc (hunk 2 above)
Comment (arjantijms) [reply src/main/antora/modules/web/pages/servlets/servlets003.adoc:28]: Also note that these scopes are just the scopes CDI defines, and which always work for beans injected into a Servlet.

Beans with other (custom) scopes can also be injected, but the developer has to be sure that the scope would indeed be active inside a Servlet.

File: src/main/antora/modules/web/pages/servlets/servlets.adoc (hunk 3 above)
Comment (BalusC) [reply src/main/antora/modules/web/pages/servlets/servlets.adoc]: There were more occurrences https://github.com/jakartaee/jakartaee-tutorial/pull/36/commits/1d0f36f13a48dd67812d40f65501b0dfabf2b7db

File: src/main/antora/modules/web/pages/servlets/servlets003.adoc (hunk 1 above)
Comment (BalusC) [reply src/main/antora/modules/web/pages/servlets/servlets003.adoc]: https://github.com/jakartaee/jakartaee-tutorial/pull/36/commits/080bea3629fe01f40eac75638f8b5a1a1c8310af

File: src/main/antora/modules/web/pages/servlets/servlets003.adoc (hunk 2 above)
Comment (BalusC) [reply src/main/antora/modules/web/pages/servlets/servlets003.adoc:28]: https://github.com/jakartaee/jakartaee-tutorial/pull/36/commits/1319cecc4f88635731a28efadd19ccde8bf6d862

**Review by kito99 — APPROVED** [review]

**Pull request merged**